udisks 2.9.0 regression: org.freedesktop.UDisks2.VolumeGroup.Delete does not clean up fstab mounts any more for deactivated LV

Downstream report: https://bugzilla.redhat.com/show_bug.cgi?id=1843454

verify/rhel-8-3
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testHiddenLuks (__main__.TestStorageHidden)

DevTools listening on ws://127.0.0.1:9955/devtools/browser/d2166943-31d9-4709-b58c-0cbb886cc870
[0603/125620.779124:ERROR:egl_util.cc(60)] Failed to load GLES library: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: cannot open shared object file: No such file or directory
[0603/125620.780863:ERROR:viz_main_impl.cc(159)] Exiting GPU process due to errors during initialization
CDP: {"source":"security","level":"error","text":"Refused to apply style from 'http://127.0.0.2:9291/cockpit/$d5420386e15718c6e1a3371cf92bbc489cc702572e6a61d3a3e8c83c602e8ac8/shell/nav.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.","timestamp":1591188987042.5142,"url":"http://127.0.0.2:9291/storage"}
CDP: {"source":"security","level":"error","text":"Refused to apply inline style because it violates the following Content Security Policy directive: \"default-src 'self' http://127.0.0.2:9291\". Either the 'unsafe-inline' keyword, a hash ('sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.\n","timestamp":1591188987223.156,"url":"http://127.0.0.2:9291/cockpit/$d5420386e15718c6e1a3371cf92bbc489cc702572e6a61d3a3e8c83c602e8ac8/shell/index.js","lineNumber":14,"stackTrace":"(minified)"}
> warning: grep: /sys/class/dmi/id/power/autosuspend_delay_ms: Input/output error

> log: 
> log: 
Traceback (most recent call last):
  File "/usr/lib64/python3.8/unittest/case.py", line 633, in _callTestMethod
    method()
  File "test/verify/check-storage-hidden", line 95, in testHiddenLuks
    self.assertEqual(m.execute("grep %s /etc/fstab || true" % mount_point_1), "")
  File "/usr/lib64/python3.8/unittest/case.py", line 912, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib64/python3.8/unittest/case.py", line 1292, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib64/python3.8/unittest/case.py", line 753, in fail
    raise self.failureException(msg)
AssertionError: 'UUID=e56cbf39-fd82-4313-9ce4-4aa227eff621[120 chars] 0\n' != ''
- UUID=e56cbf39-fd82-4313-9ce4-4aa227eff621 /run/mount1 auto noauto,x-parent=b21a7705-1cdc-44a3-92b5-b4b645539599,x-parent=YdXvs9-5jYB-Fv0d-CFeg-iwEN-CAH1-qQ0KEB 0 0

Wrote screenshot to TestStorageHidden-testHiddenLuks-rhel-8-3-127.0.0.2-2401-FAIL.png
Wrote HTML dump to TestStorageHidden-testHiddenLuks-rhel-8-3-127.0.0.2-2401-FAIL.html
Wrote JS log to TestStorageHidden-testHiddenLuks-rhel-8-3-127.0.0.2-2401-FAIL.js.log
Journal extracted to TestStorageHidden-testHiddenLuks-rhel-8-3-127.0.0.2-2401-FAIL.log.gz
# Result testHiddenLuks (__main__.TestStorageHidden) failed
# 1 TEST FAILED [57s on 1-ci-srv-05]
not ok 229 test/verify/check-storage-hidden TestStorageHidden.testHiddenLuks
```
First occurrence: 2020-06-03T12:56:57.120935 | revision 6e052b7a6e7aeb381785031f2b55280c12925b6a
Times recorded: 1
Latest occurrences:

- 2020-06-03T12:56:57.120935 | revision 6e052b7a6e7aeb381785031f2b55280c12925b6a

verify/debian-testing
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testHiddenLuks (__main__.TestStorageHidden)

DevTools listening on ws://127.0.0.1:9528/devtools/browser/2455c9b8-22e1-49d3-b7a6-99f935795e8f
[0616/054517.107959:ERROR:egl_util.cc(60)] Failed to load GLES library: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: cannot open shared object file: No such file or directory
[0616/054517.110110:ERROR:viz_main_impl.cc(159)] Exiting GPU process due to errors during initialization
CDP: {"source":"security","level":"error","text":"Refused to apply style from 'http://127.0.0.2:9391/cockpit/$bdd4e5a07c9afc87259398306f22180fe383b1a871252393c615a7d515dc3002/shell/nav.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.","timestamp":1592286318042.0188,"url":"http://127.0.0.2:9391/storage"}
CDP: {"source":"security","level":"error","text":"Refused to apply inline style because it violates the following Content Security Policy directive: \"default-src 'self' http://127.0.0.2:9391\". Either the 'unsafe-inline' keyword, a hash ('sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.\n","timestamp":1592286318186.504,"url":"http://127.0.0.2:9391/cockpit/$bdd4e5a07c9afc87259398306f22180fe383b1a871252393c615a7d515dc3002/shell/index.js","lineNumber":39,"stackTrace":"(minified)"}
> warning: grep: /sys/class/dmi/id/power/autosuspend_delay_ms: Input/output error

> log: 
> log: 
Traceback (most recent call last):
  File "/usr/lib64/python3.8/unittest/case.py", line 633, in _callTestMethod
    method()
  File "test/verify/check-storage-hidden", line 95, in testHiddenLuks
    self.assertEqual(m.execute("grep %s /etc/fstab || true" % mount_point_1), "")
  File "/usr/lib64/python3.8/unittest/case.py", line 912, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib64/python3.8/unittest/case.py", line 1292, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib64/python3.8/unittest/case.py", line 753, in fail
    raise self.failureException(msg)
AssertionError: 'UUID=c2357a57-b0f3-447c-8440-62d0d44d9239[120 chars] 0\n' != ''
- UUID=c2357a57-b0f3-447c-8440-62d0d44d9239 /run/mount1 auto noauto,x-parent=cf2679ed-9b34-4fdf-82d0-2ba0fdd69995,x-parent=QzWU0g-qyvS-QEIz-VwZr-32Ab-w6bT-AhbfUf 0 0

Wrote screenshot to TestStorageHidden-testHiddenLuks-debian-testing-127.0.0.2-2501-FAIL.png
Wrote HTML dump to TestStorageHidden-testHiddenLuks-debian-testing-127.0.0.2-2501-FAIL.html
Wrote JS log to TestStorageHidden-testHiddenLuks-debian-testing-127.0.0.2-2501-FAIL.js.log
Journal extracted to TestStorageHidden-testHiddenLuks-debian-testing-127.0.0.2-2501-FAIL.log.gz
# Result testHiddenLuks (__main__.TestStorageHidden) failed
# 1 TEST FAILED [30s on 4-cockpit-7]
not ok 229 test/verify/check-storage-hidden TestStorageHidden.testHiddenLuks
```
First occurrence: 2020-06-16T05:45:35.217895 | revision 793c2a17be26523ad2a7c9836a2be38870fa28d6
Times recorded: 1
Latest occurrences:

- 2020-06-16T05:45:35.217895 | revision 793c2a17be26523ad2a7c9836a2be38870fa28d6

verify/ubuntu-stable
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testHiddenLuks (__main__.TestStorageHidden)
[0807/084849.208122:ERROR:egl_util.cc(60)] Failed to load GLES library: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: cannot open shared object file: No such file or directory

DevTools listening on ws://127.0.0.1:9444/devtools/browser/3896850f-5f53-4a8b-8c4a-af5f5776f816
[0807/084849.211880:ERROR:viz_main_impl.cc(159)] Exiting GPU process due to errors during initialization
[0807/084849.215471:ERROR:zygote_host_impl_linux.cc(259)] Failed to adjust OOM score of renderer with pid 43143: Permission denied (13)
[0807/084849.404530:ERROR:zygote_host_impl_linux.cc(259)] Failed to adjust OOM score of renderer with pid 43181: Permission denied (13)
CDP: {"source":"security","level":"error","text":"Refused to apply style from 'http://127.0.0.2:9291/cockpit/$e99288a07fc12f89a8988ce38383c6655e3cc7d48e301bafe253cf466935d7af/shell/nav.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.","timestamp":1596790131190.348,"url":"http://127.0.0.2:9291/storage"}
> warning: grep: /sys/class/dmi/id/power/autosuspend_delay_ms: Input/output error

> log: 
> log: 
Traceback (most recent call last):
  File "/usr/lib64/python3.8/unittest/case.py", line 633, in _callTestMethod
    method()
  File "test/verify/check-storage-hidden", line 95, in testHiddenLuks
    self.assertEqual(m.execute("grep %s /etc/fstab || true" % mount_point_1), "")
  File "/usr/lib64/python3.8/unittest/case.py", line 912, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib64/python3.8/unittest/case.py", line 1292, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib64/python3.8/unittest/case.py", line 753, in fail
    raise self.failureException(msg)
AssertionError: 'UUID=583c4640-3695-40d0-9b0c-02b0c3a220af[120 chars] 0\n' != ''
- UUID=583c4640-3695-40d0-9b0c-02b0c3a220af /run/mount1 auto noauto,x-parent=e4ae2908-c978-4c02-897e-84092ce188a0,x-parent=BAMrv3-eHd1-TpIv-Dy4C-uVik-ocPS-94UlrR 0 0

Wrote screenshot to TestStorageHidden-testHiddenLuks-ubuntu-stable-127.0.0.2-2401-FAIL.png
Wrote HTML dump to TestStorageHidden-testHiddenLuks-ubuntu-stable-127.0.0.2-2401-FAIL.html
Wrote JS log to TestStorageHidden-testHiddenLuks-ubuntu-stable-127.0.0.2-2401-FAIL.js.log
Journal extracted to TestStorageHidden-testHiddenLuks-ubuntu-stable-127.0.0.2-2401-FAIL.log.gz
# Result testHiddenLuks (__main__.TestStorageHidden) failed
# 1 TEST FAILED [43s on centosci-tasks-g768s]
not ok 177 test/verify/check-storage-hidden TestStorageHidden.testHiddenLuks
```
First occurrence: 2020-08-07T08:49:12.364212 | revision 0b43a25bf994b00bd20f15cd98a06d7b2aa58328
Times recorded: 1
Latest occurrences:

- 2020-08-07T08:49:12.364212 | revision 0b43a25bf994b00bd20f15cd98a06d7b2aa58328

--- a/naughty/debian-testing/932-udisks-deactivated-lv-cleanup
+++ b/naughty/debian-testing/932-udisks-deactivated-lv-cleanup
@@ -1,4 +1,0 @@
-  File "test/verify/check-storage-hidden", line *, in testHiddenLuks
-    self.assertEqual(m.execute("grep %s /etc/fstab || true" % mount_point_1), "")
-*
-AssertionError: 'UUID=*' != ''

--- a/naughty/rhel-8/932-udisks-deactivated-lv-cleanup
+++ b/naughty/rhel-8/932-udisks-deactivated-lv-cleanup
@@ -1,4 +1,0 @@
-  File "test/verify/check-storage-hidden", line *, in testHiddenLuks
-    self.assertEqual(m.execute("grep %s /etc/fstab || true" % mount_point_1), "")
-*
-AssertionError: 'UUID=*' != ''

--- a/naughty/ubuntu-stable/932-udisks-deactivated-lv-cleanup
+++ b/naughty/ubuntu-stable/932-udisks-deactivated-lv-cleanup
@@ -1,4 +1,0 @@
-  File "test/verify/check-storage-hidden", line *, in testHiddenLuks
-    self.assertEqual(m.execute("grep %s /etc/fstab || true" % mount_point_1), "")
-*
-AssertionError: 'UUID=*' != ''

